src/js/index.js

```diff
@@ -1178,16 +1182,195 @@ function closeLoadingModal(){
     var modal = document.getElementById('side-menu');
-    modal.classList.add('show');
+
+    if (!modalMainRegLog.classList.contains('show') && !modal.classList.contains('show')){
+        modal.classList.add('show');
+    }
+    else {
+        modal.classList.remove('show');
+    }
 }
 
-// Close side menu
-function closeSideMenu(){
-    var modal = document.getElementById('side-menu');
-    modal.classList.remove('show');
+// Toggle parent modal of Register / login screen
+function toggleMainRegLogModal(){
+    var modalMainRegLog = document.getElementById('register-login');
+    if (modalMainRegLog.classList.contains('show')){
+        modalMainRegLog.classList.remove('show');
+    }
+    else {
+        modalMainRegLog.classList.add('show');
+    }
+}
+
+// Toggle form modals inside parent Register / login screen
+function changeForm(page){
+    var modalRegister = document.getElementById('register-screen');
+    var modalLogin = document.getElementById('login-screen');
+    var modalReset = document.getElementById('resetpass-screen');
+    var modalThankyou = document.getElementById('thankyou-screen');
+    var modalMainRegLog = document.getElementById('register-login');
+
+    if (!modalMainRegLog.classList.contains('show')){
+        toggleMainRegLogModal();
+    }
+
+    if (page == 'Login'){
+        if (modalRegister.classList.contains('show')){
+            modalRegister.classList.remove('show');
+        }
+
+        if (modalReset.classList.contains('show')){
+            modalReset.classList.remove('show');
+        }
+
+        modalLogin.classList.add('show');
+    }
+
+    if (page == 'Register'){
+        if (modalLogin.classList.contains('show')){
+            modalLogin.classList.remove('show');
+        }
+
+        if (modalReset.classList.contains('show')){
+            modalReset.classList.remove('show');
+        }
+
+        modalRegister.classList.add('show');
+    }
+
+    if (page == 'Reset'){
+        if (modalRegister.classList.contains('show')){
+            modalRegister.classList.remove('show');
+        }
+
+        if (modalLogin.classList.contains('show')){
+            modalLogin.classList.remove('show');
+        }
+
+        modalReset.classList.add('show');
+    }
+
+    if (page == 'Thankyou'){
+        if (modalRegister.classList.contains('show')){
+            modalRegister.classList.remove('show');
+        }
+
+        if (modalLogin.classList.contains('show')){
+            modalLogin.classList.remove('show');
+        }
+
+        modalThankyou.classList.add('show');
+    }
+}
+
+// Close thank you modal and container modal
+function closeThankyouPage(){
+    var modalThankyou = document.getElementById('thankyou-screen');
+    if (modalThankyou.classList.contains('show')){
+        modalThankyou.classList.remove('show');
+    }
+    toggleMainRegLogModal();
+}
+
+// Function that process form requests
+function submitForm(form){
```

Comment:
We could also capture the `Enter` key event as well for the form submission.

Reply:
Could you please explain a little more, your idea?

Reply:
When the user is in the login form, if the user presses enter the form should submit.

Reply:
Ohh sorry. Now i understood your point. Thanks.